File: src/lantana_data/lantana-search-query.ts
// ˅
'use strict';

import { LantanaSearchType } from './lantana-search-type';

// ˄

export class LantanaSearchQuery {
    // ˅

    // ˄

    tags: Array<string>;

    words: string;

    mood: number;

    lantana_search_type: LantanaSearchType;

    constructor() {
        // ˅
        this.tags = new Array<string>()
        this.words = ""
        this.mood = 0
        this.lantana_search_type = LantanaSearchType.all
        // ˄
    }

    // ˅

    // ˄
}

// ˅

// ˄

File: src/app/lantana/lantana_search_query.go
// ˅
package lantana

// ˄

type LantanaSearchQuery struct {
	// ˅

	// ˄

	Tags []string `json:"tags"`

	Words string `json:"words"`

	Mood int `json:"mood"`

	LantanaSearchType *LantanaSearchType `json:"lantana_search_type"`

	// ˅

	// ˄
}

// ˅

// ˄
